tests/data/playwright/report.xml

```diff
@@ -0,0 +1,170 @@
+<testsuites id="" name="" tests="78" failures="0" skipped="0" errors="0" time="13.506841">
```

Comment:
Can we add the failed test case?

Reply:
OK, I'll update the report!

Reply:
Updated https://github.com/launchableinc/cli/pull/746/commits/6e619060c440e0604871891bc5231a404423fae6